src/content/community/releases/_index.en.md

```diff
@@ -5,6 +5,14 @@ weight = 40
 +++
 <!-- markdownlint-disable no-duplicate-header -->
 
+## v0.14.4
+
+This is a bugfix release:
+
+* Fix stale IPtable rules along with globalIP leak which can sometimes happen as part of stress testing.
+* Handle out of order remote endpoint notifications properly in various Route Agent handlers.
+* Ensure that reverse path filtering setting is properly applied on the relevant network interfaces.
```

Comment:
@yboaron Can you please add this for the issue https://github.com/submariner-io/cloud-prepare/issues/667

`Fix issue with spawning gateway nodes in cloud prepare for OVN Kubernetes CNI`